Formula/tremor-runtime.rb

```diff
@@ -0,0 +1,105 @@
+class TremorRuntime < Formula
+  desc "Early-stage event processing system for unstructured data"
+  homepage "https://www.tremor.rs/"
+  url "https://github.com/tremor-rs/tremor-runtime/archive/refs/tags/v0.12.4.tar.gz"
+  sha256 "91cbe0ca5c4adda14b8456652dfaa148df9878e09dd65ac6988bb781e3df52af"
+  license "Apache-2.0"
+  head "https://github.com/tremor-rs/tremor-runtime.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "gcc"
+    depends_on "llvm"
+    depends_on "openssl@1.1"
+  end
+
+  fails_with :clang do
+    build 1200
+    cause "Requires C++20"
+  end
```

Comment:
I don't think this is correct. This will prevent `brew` from building with Apple Clang on Catalina, but that's not necessary because the Catalina build worked fine before this change.

`brew` will probably end up installing GCC, but then we'll end up with linkage to GCC runtime libraries.